### PR TITLE
Alessio thankyoupage

### LINKED ISSUE #282: Thankyou page after update payment data/upgrade
## Feature request
add thankyou page after update payment data and after upgrade or downgrade
### Feature description
create the thank you page as per mockup

### PR DESCRIPTION
**Pull request by cpalessio96**
### All Submissions:

*   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
*   [x] Does your code has proper inline documentation? 

### Changes proposed in this Pull Request:
Add thank you page for update payment data, upgrade plan and dowgrade plan.

Closes #282  .

### CODE REVIEW

File: assets/payment/settings_script.js [hunk 1]
```diff
@@ -51,6 +51,18 @@ function customStripeForm(planRank){
 	}
 	showCustomFormWindow(planRank, CurrentWindow);
 }
+//this function is used by the dialog CustomFormWindow 
+function thankYouPage(mode){
```
Comment (GiuseppeIuculano) [inline assets/payment/settings_script.js:55]: Codacy found an issue: ['thankYouPage' is defined but never used.](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5975032)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- assets/payment/settings_script.js  2
- assets/payment/payment_template.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5975032&bid=19590968)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- assets/payment/settings_script.js  2
- assets/payment/payment_template.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5975032&bid=19590968)

File: assets/payment/settings_script.js (hunk 1 above)
Comment (GiuseppeIuculano) [inline assets/payment/settings_script.js:55]: Codacy found an issue: ['thankYouPage' is defined but never used.](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5975032)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- assets/payment/settings_script.js  2
- assets/payment/payment_template.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5975032&bid=19590968)

File: assets/payment/settings_script.js (hunk 1 above)
Comment (GiuseppeIuculano) [inline assets/payment/settings_script.js:55]: Codacy found an issue: ['thankYouPage' is defined but never used.](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5975032)

**Review by HyprCoder — APPROVED** [review]

**Pull request merged**